app/lib/dttp/code_sets/age_ranges.rb

```diff
@@ -8,23 +8,19 @@ module AgeRanges
         "5 to 11 programme" => { option: :main, entity_id: "22215c74-8ec0-e611-80be-00155d010316" },
         "11 to 16 programme" => { option: :main, entity_id: "feed86a2-8ec0-e611-80be-00155d010316" },
         "11 to 19 programme" => { option: :main, entity_id: "69e663ac-8ec0-e611-80be-00155d010316" },
-        "3 - 7 programme" => { option: :additional, entity_id: "4660ff36-3c6d-e711-80d2-005056ac45bb" },
-        "3 - 8 programme" => { option: :additional, entity_id: "b86eee51-8ec0-e611-80be-00155d010316" },
-        "3 - 9 programme" => { option: :additional, entity_id: "8531bb82-2f71-e811-80f3-005056ac45bb" },
-        "0 - 5 programme" => { option: :additional, entity_id: "eea15e61-8ec0-e611-80be-00155d010316" },
-        "3 - 11 programme" => { option: :additional, entity_id: "efa15e61-8ec0-e611-80be-00155d010316" },
-        "5 - 9 programme" => { option: :additional, entity_id: "21215c74-8ec0-e611-80be-00155d010316" },
-        "5 - 11 programme" => { option: :additional, entity_id: "22215c74-8ec0-e611-80be-00155d010316" },
-        "5 - 14 programme" => { option: :additional, entity_id: "187ee9f5-569c-e711-80d9-005056ac45bb" },
-        "7 - 11 programme" => { option: :additional, entity_id: "fad38a88-8ec0-e611-80be-00155d010316" },
-        "7 - 14 programme" => { option: :additional, entity_id: "fbd38a88-8ec0-e611-80be-00155d010316" },
-        "7 - 16 programme" => { option: :additional, entity_id: "1a7ee9f5-569c-e711-80d9-005056ac45bb" },
-        "9 - 14 programme" => { option: :additional, entity_id: "7064d397-8ec0-e611-80be-00155d010316" },
-        "9 - 16 programme" => { option: :additional, entity_id: "1c7ee9f5-569c-e711-80d9-005056ac45bb" },
-        "11 - 16 programme" => { option: :additional, entity_id: "feed86a2-8ec0-e611-80be-00155d010316" },
-        "11 - 19 programme" => { option: :additional, entity_id: "69e663ac-8ec0-e611-80be-00155d010316" },
-        "14 - 19 programme" => { option: :additional, entity_id: "73a86fb7-8ec0-e611-80be-00155d010316" },
-        "14 - 19 diploma" => { option: :additional, entity_id: "41a9eb3f-5a9c-e711-80d9-005056ac45bb" },
+        "3 to 7 programme" => { option: :additional, entity_id: "4660ff36-3c6d-e711-80d2-005056ac45bb" },
+        "3 to 8 programme" => { option: :additional, entity_id: "b86eee51-8ec0-e611-80be-00155d010316" },
+        "3 to 9 programme" => { option: :additional, entity_id: "8531bb82-2f71-e811-80f3-005056ac45bb" },
+        "0 to 5 programme" => { option: :additional, entity_id: "eea15e61-8ec0-e611-80be-00155d010316" },
```

Comment:
![image](https://user-images.githubusercontent.com/470137/102895575-d66bc880-445c-11eb-81f8-187b21db6743.png)

Should we order this?